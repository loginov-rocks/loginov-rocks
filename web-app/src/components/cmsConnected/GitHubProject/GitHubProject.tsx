import * as React from 'react';

import { WorksListItem } from 'components/contextConnected/WorksListItem';

interface Props {
  gitHubId: string;
}

export const GitHubProject: React.FC<Props> = ({ gitHubId }) => (
  <WorksListItem gitHubId={gitHubId} />
);
