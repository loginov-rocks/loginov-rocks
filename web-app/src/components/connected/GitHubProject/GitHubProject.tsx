import * as React from 'react';

import { WorksListItem } from 'components/smart/WorksListItem';

interface Props {
  gitHubId: string;
}

export const GitHubProject: React.FC<Props> = ({ gitHubId }) => (
  <WorksListItem gitHubId={gitHubId} />
);
