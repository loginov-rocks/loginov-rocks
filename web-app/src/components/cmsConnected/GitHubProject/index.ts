import { GitHubProject } from './GitHubProject';

export { GitHubProject };
