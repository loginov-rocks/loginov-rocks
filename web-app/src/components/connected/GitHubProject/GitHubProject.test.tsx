/* eslint-disable import/no-extraneous-dependencies */

import { shallow } from 'enzyme';
import * as React from 'react';

import { GitHubProject } from './GitHubProject';

it('matches snapshot', () => {
  const wrapper = shallow(
    <GitHubProject
      gitHubId="Web-Bluetooth-Terminal"
    />,
  );

  expect(wrapper).toMatchSnapshot();
});
