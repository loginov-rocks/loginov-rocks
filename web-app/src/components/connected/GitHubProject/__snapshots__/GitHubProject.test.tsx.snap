// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<WorksListItem
  gitHubId="Web-Bluetooth-Terminal"
/>
`;
